ResearchKit/Localized/en.lproj/Localizable.strings

```diff
@@ -86,6 +86,10 @@
 /* Alert for invalid email address value */
 "INVALID_EMAIL_ALERT_MESSAGE" = "Invalid email address: %@";
 
+/* Alerts for invalid location coordinates */
+"LOCATION_ADDRESS" = "Enter an address";
+"LOCATION_ERROR_TITLE" = "Location error";
```

Comment:
Or we just the original `"Invalid value"`? An `Unresolvable address` is indeed an invalid value for this type question.
What do you think?